packages/config/src/bridges/hop.ts

```diff
@@ -203,4 +204,21 @@ export const hop: Bridge = {
       description: 'Participants of the 2/3 Hop MultiSig.',
     },
   ],
+  knowledgeNuggets: [
+    {
+      title: 'Hop protocol deep dive',
+      url: 'https://li.fi/knowledge-hub/hop-a-deep-dive/',
+      thumbnailUrl: NUGGETS.LIFI_THUMBNAIL,
+    },
+    {
+      title: 'How Hop protocol works?',
+      url: 'https://twitter.com/bkiepuszewski/status/1437320613358673922',
+      thumbnailUrl: NUGGETS.BARTEK_TWITTER_THUMBNAIL
+    },
+    {
+      title: 'How HTLC bridge works?',
+      url: 'https://twitter.com/bkiepuszewski/status/1437031523455229964',
+      thumbnailUrl: NUGGETS.BARTEK_TWITTER_THUMBNAIL
+    },
+  ],
```

Comment:
Let's maybe agree on a consistant sort of the nuggets? I'm compraing it to cBridge here

Reply:
Let's postpone this conversation for the next iteration